src/Features/LanguageServer/Protocol/Features/UnifiedSuggestions/UnifiedSuggestedActionsSource.CodeRefactoringsComputer.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.FixAll;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+using static Microsoft.CodeAnalysis.CodeActions.CodeAction;
+
+namespace Microsoft.CodeAnalysis.UnifiedSuggestions
+{
+    /// <summary>
+    /// Provides mutual code action logic for both local and LSP scenarios
+    /// via intermediate interface <see cref="IUnifiedSuggestedAction"/>.
+    /// </summary>
+    internal partial class UnifiedSuggestedActionsSource
+    {
+        private static class CodeRefactoringsComputer
```

Comment:
Most of the code in this file is existing code split out from `UnifiedSuggestedActionsSource.cs`. The only new code is `GetUnifiedFixAllSuggestedActionSetAsync` towards the end of this file.

Reply:
ok.  i'm trusting you on this :)